docs/release-notes/march-2024.md

```diff
@@ -0,0 +1,119 @@
+- Users can see all projects, even if they don't have any identified findings, in Semgrep Cloud Platform.
+- Semgrep Code now distinguishes between findings resolved by rule changes and findings resolved due to code modifications. This change applies only to new findings.
+  - Only findings fixed due to code modifications are marked as fixed.
+    - The fix rate calculated by Semgrep Code now includes only such findings.
+  - Findings fixed due to rule changes are marked as resolved.
+- **CLI**: Semgrep clones the repository into the current working directory instead of a `tmp` folder when using the `-- remote` flag.
+
+### Fixed
+
+- Kotlin: Fixed a parsing error when a newline appears between the class name and the primary constructor.
+- Fixed an issue where autofix on variable definitions could not handle semicolons for Java, C++, C#, Rust, Cairo, Solidity, and Dart.
+- Fixed an issue with autofix application on lines with multi-byte characters.
+- Fixed issue where credentials were inadvertently included in a project URL when publishing a custom rule using `semgrep publish`. Running `semgrep publish` generates a `rule-origin-note`, which includes the project URL in the metadata. When this process occurs in a GitLab CI job, GitLab includes the CI job tokens in the project URL. Semgrep now removes the credential from the metadata.
+- Fixed an issue where reachability rules were deleted from Semgrep Registry.
+- Fixed an issue where the timestamp on the findings didn't correspond to the timestamp used by the filter; now, both use the `relevant_since` filter, which provides information about when findings were last reopened.
+
+## ⛓️ Supply Chain
+
+### Added
+
+- Supply Chain now offers lockfile-only support for Swift projects.
+- Added a tour of Supply Chain features in Semgrep Cloud Platform for first-time users, as well as a tour for returning users.
+
+### Changed
+
+- Findings with a critical severity now display in Semgrep Cloud Platform with a darker red color to help distinguish them from high-severity findings.
+- Findings are now displayed in Semgrep Cloud Platform with readable names, such as `git-url-parse: Inefficient Regular Expression Complexity` instead of `lodash.defaultsdeep: Improper Input Validation`.
+- Added additional reachability filter values to Semgrep Cloud Platform. Users can now search using the following values: **Reachable**, **Always Reachable**, **Conditionally Reachable**, **Unreachable**, **Unknown**.
+
+### Fixed
+
+- Fixed an issue where reachable findings were labeled **Unreachable** in Semgrep Cloud Platform.
+- Fixed an issue where bulk triage didn't work in Semgrep Cloud Platform for Supply Chain findings.
+- Fixed an issue where Supply Chain rules and findings erroneously display a confidence label.
+
+## 🤖 Assistant (beta)
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 🔐 Secrets
+
+Semgrep Secrets is no longer beta. It is now generally available.
```

Comment:
```suggestion
Semgrep Secrets is now generally available.
```
Though - wait a sec - I thought it wasn't yet? [Slack thread](https://semgrepinc.slack.com/archives/C04L9V5NDPG/p1710439729240409)

Reply:
I see that we removed the beta label, but I don't see a corresponding blog post? We usually have an announcement about it, right?

Reply:
So we've found that odd in-between, [where secrets is GA but marketing is forthcoming](https://semgrepinc.slack.com/archives/C010CKRBLN9/p1709151677609539)

Reply:
Let's remove the bullet for now.